account_payment_order/wizard/res_config_settings.py

```diff
@@ -10,3 +10,7 @@ class ResConfigSettings(models.TransientModel):
     transfer_journal_id = fields.Many2one(
         related="company_id.transfer_journal_id", readonly=False
     )
+    use_allow_out_payment = fields.Boolean(
+        string=" Use allow out payment on partner bank",
```

Comment:
```suggestion
        string="Verify that Send Money is enabled on partner bank accounts",
```

Is this easier to understand by non technical users?